src/opportunities/opportunities.js

```diff
@@ -16,6 +16,11 @@ import { AuditBuilder } from '../common/audit-builder.js';
 import { getRUMDomainkey } from '../support/utils.js';
 
 const DAYS = 30;
+const OPPTY_QUERIES = [
```

Comment:
since there can be other opportunities, which are not experimentation related, so I've kept those as `experimentationHandlers`, so can we still keep the word `experimentation`? like `EXPERIMENTATION_OPPTY_QUERIES` ?

Reply:
Actually, more accurate name for this audit should be `experimentation-opportunities` because it only encompasses the experimentation opportunities. Other type opportunities such as broken backlinks or cwv etc. are covered in different audits. Consecutively, I renamed the audit